addons/lunch/models/lunch_alert.py

```diff
@@ -37,13 +37,13 @@ class LunchAlert(models.Model):
     tz = fields.Selection(_tz_get, string='Timezone', required=True, default=lambda self: self.env.user.tz or 'UTC')
 
     until = fields.Date('Show Until')
-    recurrency_monday = fields.Boolean('Monday', default=True)
-    recurrency_tuesday = fields.Boolean('Tuesday', default=True)
-    recurrency_wednesday = fields.Boolean('Wednesday', default=True)
-    recurrency_thursday = fields.Boolean('Thursday', default=True)
-    recurrency_friday = fields.Boolean('Friday', default=True)
-    recurrency_saturday = fields.Boolean('Saturday', default=True)
-    recurrency_sunday = fields.Boolean('Sunday', default=True)
+    mon = fields.Boolean('Mon', default=True)
+    tue = fields.Boolean('Tue', default=True)
+    wed = fields.Boolean('Wed', default=True)
+    thu = fields.Boolean('Thu', default=True)
+    fri = fields.Boolean('Fri', default=True)
+    sat = fields.Boolean('Sat', default=True)
+    sun = fields.Boolean('Sun', default=True)
```

Comment:
Do you really need these strings here ? You could simply have the default=True

Reply:
@arj-odoo Agree.